### ISSUE TITLE
Issue with 2.0.202.20

### ISSUE THREAD

**Issue by oschwab** [egocarib/CavesOfQud-QudUX-v2#4]
Hi,

Qud devs modified ConsoleChar in latest beta : 

```
202.20 - 'beta' branch

    [modding] ConsoleChar underwent a large internal refactor to simplify its data structure and provide a more flexible framework for colors.

```

So QudUX launch is broken at startup : 

```
WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(22,13): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(24,13): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(26,13): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(28,13): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(33,21): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(35,21): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(40,21): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\ConsoleUtilities.cs(42,21): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

WARN - Mod Warning QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\TileMaker.cs(69,13): warning CS0618: 'ConsoleChar.Attributes' is obsolete: 'Don't use this. Will be removed ~Q2 2021' 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

MODERROR  - Mod Error QudUX: <...>\steamapps\workshop\content\333640\1804499742\Utilities\TileMaker.cs(72,35): error CS1061: 'ConsoleChar' does not contain a definition for 'TileLayerBackground' and no accessible extension method 'TileLayerBackground' accepting a first argument of type 'ConsoleChar' could be found (are you missing a using directive or an assembly reference?) 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)
``

I'll try to have a look to it but I'm quite busy with work these days so I can't be sure.

**Comment by oschwab** [comment]
PR done , see https://github.com/egocarib/CavesOfQud-QudUX-v2/pull/5

**Comment by egocarib** [comment]
For now, this is fixed by 893075e7f8b7c75e203f78fb1829c1b56ec44a0b.

Eventually I will switch over to a simpler fix like the one you have proposed (after the ConsoleChar changes merge into stable)
